.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 # R specific hooks: https://github.com/lorenzwalthert/precommit
 repos:
 -   repo: https://github.com/lorenzwalthert/precommit
-    rev: v0.3.2.9021
```

Comment:
Please undo this commit (`git reset 27f5bdee5a8cb4d37593d077a5960dbfc94ad063; git push -f`), the versions are bumped in a different process.